examples/using-babel-typescript/babel.config.js

```diff
@@ -0,0 +1,27 @@
+module.exports = {
```

Comment:
Verified that it works also with `babel.config.js`. Should we rename this because `'cypress-react-unit-test/plugins/babelrc'` looks confusing.

Reply:
sure, rename it

Reply:
Sorry, I've been not clear. I mean the name of plugin itself. It will require breaking change, right? Or just make a new alias name?

Reply:
right - ok, in that case we probably better with an alias, like react-scripts is an alias to car.